Documentation-Requirements.md

```diff
@@ -0,0 +1,60 @@
+* [Introduction](#introduction)
+* [General requirements](#general-requirements)
+* [Code blocks](#code-blocks)
+
+# Introduction
+
+This document outlines the requirements for all documentation in the [Kata
+Containers](https://github.com/kata-containers) project.
+
+# General requirements
+
+- All documents are expected to be written in GitHub Flavored Markdown format.
+- All documents should have the `.md` file extension.
+
+# Code blocks
+
+This section lists requirements for displaying commands and command output.
+
+The requirements must be adhered to since documentation containing code blocks
+is validated by the CI system which executes the command blocks with the help
```

Comment:
missing a comma after "system": is validated by the CI system, which executes the command blocks with the help

Reply:
So close! :smile: Branch updated.